src/components/home/About.jsx

```diff
@@ -1,5 +1,50 @@
+import React from "react";
+import Title from "../Title";
+import Subtitle from "../Subtitle";
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="p-56 mx-96 flex flex-col items-center">
+      <div className="text-center">
+        <Title subTitle="LEARN WHO WE ARE" title="ABOUT US"></Title>
+      </div>
+
+      <img
+        className="my-4 flex justify-center mx-auto"
+        src={"assets/fullLogo.webp"}
+        alt="Engineers Without Borders Logo"
+      />
+      <div className="text-justify  col-span-3 items-center">
+        Founded in 2007, the University of California, Riverside Chapter of
+        Engineers Without Borders is part of the Inland Empire Chapter (EWB-IE,
+        along with Cal Poly Pomona and the Inland Empire professional chapter).
+        We are dedicated to serving the UCR campus, the City of Riverside, and
+        any community across the world that needs our technical expertise.
+      </div>
+      <img
+        className="my-4 flex justify-center"
+        src={"images/AboutPic1.webp"}
+        alt="Community of Riverside"
+      />
+      <div className="w-full text-left my-3">
```

Comment:
if you remove the flex flex-col items-center in the parent div, you can remove this div